docs/api/app.md

```
@@ -761,3 +761,56 @@ func main() {
+
+
+## RemoveRoute
+
+This method removes a route by path.  You must call the `RebuildTree()` method after the remove in to ensure the route is removed.
+
+```go title="Signature"
+func (app *App) RemoveRoute()
+```
+
+This method removes a route by name
+```go title="Signature"
```

Markdownlint (0.37.0)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)